test_conformance/vulkan/test_vulkan_interop_buffer.cpp

```diff
@@ -742,9 +742,9 @@ int run_test_with_multi_import_same_ctx(
     }
     else
     {
-        clVk2CLExternalSemaphore = new clExternalSemaphore(
+        clVk2CLExternalSemaphore = new clExternalImportableSemaphore(
             vkVk2CLSemaphore, context, vkExternalSemaphoreHandleType, deviceId);
-        clCl2VkExternalSemaphore = new clExternalSemaphore(
+        clCl2VkExternalSemaphore = new clExternalExportableSemaphore(
             vkCl2VkSemaphore, context, vkExternalSemaphoreHandleType, deviceId);
```

Comment:
This doesn't necessarily be an exportable semaphore for other handle types. Pretty much all semaphores can be created by Vulkan, imported by OpenCL and can be wait on or signaled from either APIs.

Reply:
It does need to be exportable.  It is passed a Vulkan semaphore in the function signature, and is then signaled on from OpenCL later.  The original code takes a Vulkan semaphore, so it always did an import or export.  The spec now restricts simultaneous importing and export, so all external semaphores must be importable or exportable.  There is no other option.